docs/guides/api-keys.md

```diff
@@ -32,16 +32,16 @@ const { SuperfaceClient } = require('@superfaceai/one-sdk');
 const sdk = new SuperfaceClient();
 
 async function run() {
-  // Load the capability profile
+  // Load the profile
   const profile = await sdk.getProfile('communication/send-email@2.1.0');
 
   // Get provider and configure SendGrid token
   const provider = client.getProvider(
-	'sendgrid', {
-		security: {
-			['bearer_token']: { token: '<your-token-from-sendgrid>' }
-		}
-	});
+    'sendgrid', {
+    security: {
+      ['bearer_token']: { token: '<your-token-from-sendgrid>' }
```

Comment:
Why do you need `[]` here? Won't just this work?

```suggestion
      bearer_token: { token: '<your-token-from-sendgrid>' }
```

(I assume this could be because ESLint complaining about snake_case but that's not issue of docs.)

Reply:
fixed